src/java.base/share/classes/com/sun/crypto/provider/SunJCE.java

```diff
@@ -235,16 +229,31 @@ void putEntries() {
         psA("Cipher", "AES_256/CFB/NoPadding",
                 "com.sun.crypto.provider.AESCipher$AES256_CFB_NoPadding",
                 attrs);
+
+        attrs.clear();
+        attrs.put("SupportedModes", "GCM");
+        attrs.put("SupportedKeyFormats", "RAW");
+
+//        ps("Cipher", "GCM",
+//            "com.sun.crypto.provider.GaloisCounterMode$AESGCM", null, attrs);
+        ps("Cipher", "AES/GCM/NoPadding",
+            "com.sun.crypto.provider.GaloisCounterMode$AESGCM", null, attrs);
```

Comment:
It's part of the getInstance() provider selection.  in AESCipher it does the same thing with CBC, ECB, etc, but the AESCipher.General is used.  Because this is the only AES mode specified that is not using General and outside AESCipher, it had to be implemented separately